psalm.xml.dist

```diff
@@ -26,6 +26,14 @@
                 <referencedMethod name="Roave\PsrContainerDoctrine\AbstractFactory::retrieveConfig"/>
                 <referencedMethod name="Roave\PsrContainerDoctrine\AbstractFactory::__callStatic"/>
                 <referencedMethod name="Roave\PsrContainerDoctrine\AbstractFactory::__invoke"/>
+
+                <!-- InvocationMocker is still internal. See https://github.com/sebastianbergmann/phpunit/issues/3742 -->
```

Comment:
Does adding `psalm/plugin-phpunit` fix these?

Reply:
It's already enabled.

```
$ vendor/bin/psalm-plugin enable psalm/plugin-phpunit
 ! [NOTE] Plugin already enabled
```

Reply:
Ah okie dokie :)